code/modules/surgery/organs/blood.dm

```diff
@@ -95,9 +95,11 @@
 		..()
 		if(dna.species.exotic_blood)
 			var/datum/reagent/R = GLOB.chemical_reagents_list[get_blood_id()]
-			if(istype(R) && isturf(loc))
-				R.reaction_turf(get_turf(src), amt * EXOTIC_BLEED_MULTIPLIER)
-
+			if(EXOTIC_COLOR in dna.species.species_traits)
```

Comment:
isn't this a bit double? Since it's already checked on line 96

Reply:
Resolved over discord